com.unity.render-pipelines.core/Runtime/Debugging/DebugUI.Fields.cs

```diff
@@ -241,6 +241,39 @@ public override float ValidateValue(float value)
             }
         }
 
+        static class EnumUtility
+        {
+            internal static GUIContent[] MakeEnumNames(Type enumType)
+            {
+                return enumType.GetFields(BindingFlags.Public | BindingFlags.Static).Select(fieldInfo =>
+                {
+                    var description = fieldInfo.GetCustomAttributes(typeof(InspectorNameAttribute), false);
+
+                    if (description.Length > 0)
+                    {
+                        return new GUIContent(((InspectorNameAttribute)description.First()).displayName);
+                    }
+
+                    // Space-delimit PascalCase (https://stackoverflow.com/questions/155303/net-how-can-you-split-a-caps-delimited-string-into-an-array)
+                    var niceName = Regex.Replace(fieldInfo.Name, "([a-z](?=[A-Z])|[A-Z](?=[A-Z][a-z]))", "$1 ");
+                    return new GUIContent(niceName);
+                }).ToArray();
+            }
+
+            internal static int[] MakeEnumValues(Type enumType)
+            {
+                // Linq.Cast<T> on a typeless Array breaks the JIT on PS4/Mono so we have to do it manually
+                //enumValues = Enum.GetValues(value).Cast<int>().ToArray();
+
+                var values = Enum.GetValues(enumType);
+                var enumValues = new int[values.Length];
+                for (int i = 0; i < values.Length; i++)
+                    enumValues[i] = (int)values.GetValue(i);
+
+                return enumValues;
+            }
+        }
```

Comment:
I think this code is pretty specific to debug views. I'm skipping `ObsoleteAttribute` because debug modes are never serialized, so there's no need to mark anything obsolete - it will just be removed. For general-purpose inspector enums, there is already `EnumDataUtility`.